tests/rustdoc-ui/2024-hashtag-doctest.rs

```diff
@@ -0,0 +1,14 @@
+// This test ensures that `##` are not emitting a warning when generating
+// docs with the 2024 edition.
+// Regression test for <https://github.com/rust-lang/rust/issues/136899>.
+
+//@ check-pass
+
+#![deny(warnings)]
+
+//! Test
+//!
+//! ```
```

Comment:
Alternatively


```suggestion
//! ```edition2024
```

Reply:
Since we want to test both editions, I'll just use revisions instead. :)